r3bsource/sync_check/R3BSyncCheckReader.cxx

```diff
@@ -54,22 +54,40 @@ Bool_t R3BSyncCheckReader::Init(ext_data_struct_info* a_struct_info)
         return kFALSE;
     }
     // Register output array in tree
-    FairRootManager::Instance()->Register("SyncCheckData", "SyncCheck", fArray, kTRUE);
+    FairRootManager::Instance()->Register("SyncCheckData", "SyncCheck", fArray, kFALSE);
```

Comment:
This means that you don't want to store the sync info in the output root file, is this right?

Reply:
Indeed, I do not consider this data to be important in the output tree. It is primarily required for the online monitor.  Anyway, I have just added a flag and a setter to store it optionally (see my last commit)